### ISSUE TITLE
Auto Complete Doesn't Work With Multiple Words

### ISSUE THREAD

**Issue by McCrearyD** [masterT/ultimate-guitar-scraper#29]
If you try to do something like https://www.ultimate-guitar.com/static/article/suggestions/w/wish%20you.js it will come back with nothing. ( clicking gives you 404 ). I've tried substituting the space with other things like +, &, -, and many others but no luck. Any ideas?

**Comment by gallowdp** [comment]
I'm not the owner of this repo but I've been using it and just did some digging on the site. It appears that the UG API only allows a max of 5 characters for its autocomplete query. It does use underscores for spaces below the 5 character limit, but after 5 characters, it just keeps hitting those 5 characters to the autocomplete API. I've attached a screenshot of Chrome devtools using the Network tab to show what's happening behind the scenes. It looks like you'd want to add a 5-char limit to your app as well to avoid hitting errors. Hope this helps!

![image](https://user-images.githubusercontent.com/2529581/39670072-f5655c18-50b9-11e8-9284-5f29c1beb3d6.png)

**Comment by gallowdp** [comment]
Example that works with a space (underscore): https://www.ultimate-guitar.com/static/article/suggestions/t/the_t.js

**Comment by masterT** [comment]
Hi, 
it looks like Ultimate Guitar change their suggestion endpoint. I would be happy if you want to fix this issue (replace space with `_`)!

**Comment by McCrearyD** [comment]
Good find, I've never used the network console. Thank you @gallowdp :)

**Comment by gallowdp** [comment]
Glad I could help! It does look like some additional filtering is done on the front-end after the max of 5 characters are entered. Even though it sends a max of 5 characters to the endpoint, UG's front-end autocomplete interface still filters after entering more characters. I think this logic can also just be implemented on the front-end of whatever app is uses this scraper, but it could be an enhancement to incorporate in this library.

**Comment by McCrearyD** [comment]
@gallowdp true, yeah. currently working on a tablature parser actually, and incorporating this into it if you wanna check it out. takes the raw text and turns it into json so you can format it more graphically. you can contribute if you’d like.

**Comment by masterT** [comment]
You should treat this scraper as an API client, maybe refine the results with `Array.prototype.filter` if you need to.

**Comment by masterT** [comment]
Thanks for the help! I just publish `ultimate-guitar-scraper@1.0.2` 🎉 !

**Comment by McCrearyD** [comment]
@masterT glad i could contribute! and it’s a nice piece you made, will definitely let you know if i find anything else.
